website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx

```diff
@@ -129,27 +133,46 @@ To create a policy set:
+   - **Enhanced:** A Sentinel policy set uses a [policy evaluation](#policy-evaluations) in Terraform Cloud. This lets you enable policy overrides and enforce a Sentinel runtime version
+1. **(OPA Only)** Select a **Runtime version** for this policy set.
+
+    -> **Note**: Policy runtime version management is in beta.
 
-1. **(Optional)** Add **Policy exclusions** for this policy set. Specify any workspaces in the policy set's scope that Terraform Cloud will not enforce this policy set on.
 1. **(OPA Only)** Allow **Overrides**, which enables users with override policy permissions to apply plans that have [mandatory policy](#policy-enforcement-levels) failures.
 1. **(VCS Only)** Optionally specify the **VCS branch** within your VCS repository where Terraform Cloud should import new versions of policies. If you do not set this field, Terraform Cloud uses your selected VCS repository's default branch.
 1. **(VCS Only)** Specify where your policy set files live using the **Policies path**. This lets you maintain multiple policy sets within a single repository. Use a relative path from your root directory to the directory that contains either the `sentinel.hcl` (Sentinel) or `policies.hcl` (OPA) configuration files. If you do not set this field, Terraform Cloud uses the repository's root directory.
 1. **(Managed Policies Only)** Select managed **Policies** to add to the policy set. You can only add policies written with the same policy framework you selected for this set.
 1. Choose a descriptive and unique **Name** for the policy set. You can use any combination of letters, numbers, `-`, and `_`.
 1. Write an optional **Description** that tells other users about the purpose of the policy set and what it contains.
 
-### Edit Policy Sets
+### Edit policy sets
 
 To edit a policy set:
 1. Go to the **Policy Sets** section of your organization’s settings.
 1. Click the policy set you want to edit to go to its settings page.
 1. Adjust the settings and click **Update policy set**.
 
-### Delete Policy Sets
+### Evaluate a policy runtime upgrade
+
+You can validate that changing a policy runtime version does not introduce any breaking changes.
+
+-> **Note:** On-demand policy evaluation is in beta.
+
+To perform a policy evaluation:
+
+1. Go to the **Policy Sets** section of your organization’s settings.
+1. Click the policy set you want to upgrade.
+1. Click the **Evaluate** tab.
+1. Select the **Runtime version** you wish to upgrade to.
+1. Select a **Workspace** to test the policy and upgraded version against.
+1. Click **Evaluate**.
+
+Terraform Cloud will execute the policy set using the specified version and the latest plan data for the selected workspace. It will display the evaluation results. If the evaluation has an `Errored` status, inspect the JSON output to determine whether the issue is related to a non-compliant resource or is due to a syntax issue within a policy.
```

Comment:
```suggestion
Terraform Cloud will execute the policy set using the specified version and the latest plan data for the selected workspace. It will display the evaluation results. If the evaluation returns a `Failed` status, inspect the JSON output to determine whether the issue is related to a non-compliant resource or is due to a syntax issue within a policy.
```